### PR TITLE
[PDR-386] Add covid_19_serology_results module consent.

### PR DESCRIPTION
**Pull request by robabram**
## Description of changes/additions
* PDR user requested COVID serology consent answers.  Add answers to participant module data.

## Tests
- [x] unit tests

### CODE REVIEW

**Review by pegbertsch — COMMENTED** [review]
Do you think you could add a new unit test case based on test_questionnaire_response_dao.test_cope_first_minute_survey()?  That test makes use of the PDRGeneratorTestMixin to verify the PDR data after populating a questionnaire response payload.

**Comment by robabram** [comment]
I have added a unittest for these changes.

File: tests/test-data/questionnaire_covid_19_serology_results.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
+{
```
Comment (pegbertsch) [inline tests/test-data/questionnaire_covid_19_serology_results.json:1]: For future reference:  I think some of the more recent test infrastructure allows for dynamic generation of simulated module responses so we don't need to always create new JSON files.   E.g., using `test_questionnaire_response_dao._submit_questionnaire_response` to generate test payloads (used in a lot of the various COPE survey unit tests).  May need to know the code ID of the consent question to use.

File: tests/dao_tests/test_questionnaire_response_dao.py [hunk 2]
```diff
@@ -867,6 +869,32 @@ def test_february_cope_survey(self):
...
+        response = self.send_post(url, resource)
+        self.assertEqual(response["group"]["question"][0]["answer"][0]["valueCoding"]['code'], 'Decision_No')
+
+        record: ResourceData = self.session.query(ResourceData).\
+                filter(ResourceData.resourcePKAltID==participant_id).one()
+        decision_found = False
+        for mod in record.resource['modules']:
+            if mod['module'] == 'covid_19_serology_results':
+                decision_found = True
+                self.assertEqual(mod['consent_value'], 'Decision_No')
```
Comment (pegbertsch) [inline tests/dao_tests/test_questionnaire_response_dao.py:894]: Should it also verify the `mod_status` (maybe just `status` in the resource generator?)  is SUBMITTED_NO_CONSENT?

**Review by pegbertsch — COMMENTED** [review]
(Minor) Request to also validate the correct submittal status was assigned in the unit test generated resource data.

File: tests/dao_tests/test_questionnaire_response_dao.py (hunk 2 above)
Comment (robabram) [reply tests/dao_tests/test_questionnaire_response_dao.py:894]: Done.

File: tests/test-data/questionnaire_covid_19_serology_results.json (hunk 1 above)
Comment (robabram) [reply tests/test-data/questionnaire_covid_19_serology_results.json:1]: Sorry, I just went with the way I know for this one.

**Review by pegbertsch — APPROVED** [review]

File: tests/test-data/questionnaire_covid_19_serology_results.json (hunk 1 above)
Comment (pegbertsch) [reply tests/test-data/questionnaire_covid_19_serology_results.json:1]: np, just thought it might make adding additional tests down the line easier

**Review by wangy70 — APPROVED** [review]

**Pull request merged**